packages/devtools-core/src/App/DevTools/index.tsx

```diff
@@ -53,13 +53,52 @@ const StyledPanel = styled.div`
   max-height: calc(100vh - ${HeaderHeight});
 `;
 
+const RecordButtons = styled.div`
+  position: absolute;
+  bottom: 5px;
+  right: 5px;
+`;
+
+function post({ func, universe, actionLog }: any) {
+  const name = prompt("Name:");
+  // @ts-ignore
+  const { port } = window.devtoolsServer;
+  fetch(`http://localhost:${port}/${func}`, {
+    method: "post",
+    headers: { "Content-Type": "application/json" },
+    body: JSON.stringify({ universe, actionLog, name }),
+  })
+    .then(res => res.json())
+    .then(res => console.log(res));
+}
+
+const ServerActions = () => {
+  // @ts-ignore
+  const { buttons } = window.devtoolsServer;
+  console.log({ ServerActions, buttons });
```

Comment:
I assume these console logs are for debug, they'll have to go before it can be merged :)